### PR TITLE
Fix part of #3242: Hi-Fi User Profile Tablet 

### LINKED ISSUE #3242: Hi-Fi User Profile Tablet
Mobile - https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/1c3f250f-32e5-4299-8187-75d7c3dbe6b6/

Tablet - https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/e277771d-14bf-48a2-9974-7d48f25d3f28/

**Follow this guide**: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fixes part of  #3242 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout-land/profile_edit_fragment.xml [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+        android:layout_height="1dp"
+        android:layout_marginBottom="@dimen/profile_edit_delete_margin_bottom"
+        android:background="@color/grey"
+        android:visibility="@{viewModel.profile.isAdmin ? View.GONE : View.VISIBLE}"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+    </androidx.constraintlayout.widget.ConstraintLayout>
+  </ScrollView>
+</layout>
```
Comment (anandwana001) [inline app/src/main/res/layout-land/profile_edit_fragment.xml]: add new line

**Review by rt4914 — CHANGES_REQUESTED** [review]
Blocking on your TODO.

**Comment by rt4914** [comment]
> Blocking on your TODO.

@anandwana001 Convert the profile deletion alert dialog to `DialogFragment` similar to `ExitProfileDialogFragment` and that should persist during config change.

**Comment by rt4914** [comment]
@anandwana001 In `ProfileEditActivityTest ` tests are failing on CI.

**Comment by anandwana001** [comment]
> @anandwana001 In `ProfileEditActivityTest ` tests are failing on CI.

missed to update the id, all are passing now. PTAL

**Review by rt4914 — APPROVED** [review]
LGTM

**Pull request merged**